### ISSUE TITLE
Possibility of reverting the garbage new video layout?

### ISSUE THREAD

**Issue by KingGamingYT** [omnidevZero/YouTubeRedux#31]
I know this is technically a duplicate of #10, however, I wanted to bring it up again.

So, I've finally contracted this hideous interface change youtube is testing, and I was wondering if there's any chance of a feature being added or someone who's made a script to get rid of this eyesore. It screws up not only the layout but also breaks the icons too (like and dislike for example).
![image](https://user-images.githubusercontent.com/45918062/168194418-f148ea40-6c02-490c-a983-58d1b6cfec98.png)

**Comment by omnidevZero** [comment]
This looks slightly different than the mentioned duplicate though. Is there any way to force this one? I was kind of waiting for the previous one to appear for me before trying to fix it but in the end I never got that update so the issue dropped off my radar.

So as soon as I get that change I'll do something about it! Meanwhile I'll get back to the previous issue and I'll work with removing the attributes to force that other layout .

**Comment by omnidevZero** [comment]
Ooooh so it's the same layout! Thought it was another variant of the previous one and I was worried it could have different html despite looking similar. Thanks for that, I'll get to fixing it soon!

**Comment by KingGamingYT** [comment]
That does look a whole lot better, I just don't understand why this new interface makes descriptions look hideous. Who wants that next to the title? Even on mobile it's below the video info.

**Comment by omnidevZero** [comment]
I made a couple of changes in 2.12.3 to make video info sections feel more familiar. It's pretty much just visual changes as properly fixing everything now didn't look too feasible considering I would have to maintain separate code for people with and without that update and there are also a couple of other settings that would have to be split as well. I'll pick it up again to restore the video page to its former glory after the change has been rolled out to all users.

Closing this one. Further progress will be tracked in #10.
